### PR TITLE
Adapter Name Case Insensitive: ext.prebid.multibid

### PR DESCRIPTION
**Pull request by guscarreon**
Normalization was done in `buildMultiBidMap(prebid *openrtb_ext.ExtRequestPrebid)` inside file `exchange/exchange.go`. Let me know your thoughts.

### CODE REVIEW

File: exchange/exchange_test.go [hunk 1]
```diff
@@ -5750,3 +5750,182 @@ func TestSetSeatNonBid(t *testing.T) {
...
+				{
+					desc: "Lowercase prebid.MultiBid.Bidder is found in the BidderName list, entry is mapped",
+					inPrebid: &openrtb_ext.ExtRequestPrebid{
+						MultiBid: []*openrtb_ext.ExtMultiBid{
+							{Bidder: "appnexus"},
+						},
+					},
+					expected: map[string]openrtb_ext.ExtMultiBid{
+						"appnexus": {Bidder: "appnexus"},
+					},
```
Comment (AlexBVolcy) [inline exchange/exchange_test.go:5798]: Could you add another test case just like this, except the Bidder is capitalized: (i.e APPNEXUS)

File: exchange/exchange_test.go [hunk 2]
```diff
@@ -5750,3 +5750,182 @@ func TestSetSeatNonBid(t *testing.T) {
 		})
 	}
 }
+
+func TestBuildMultiBidMap(t *testing.T) {
```
Comment (AlexBVolcy) [inline exchange/exchange_test.go:5754]: Could you also a JSON test in `/exchangetest` similar to `multi-bids.json`, where we can test the case insensitivity there as well.

**Review by AlexBVolcy — COMMENTED** [review]
Looks good, just left comments regarding tests

File: exchange/exchange_test.go (hunk 1 above)
Comment (guscarreon) [reply exchange/exchange_test.go:5798]: Done

File: exchange/exchange_test.go [hunk 3]
```diff
@@ -5750,3 +5750,182 @@ func TestSetSeatNonBid(t *testing.T) {
...
+						},
+					},
+				},
+			},
+		},
+	}
+	for _, groups := range testGroups {
+		for _, tc := range groups.tests {
+			multiBidMap := buildMultiBidMap(tc.inPrebid)
+			assert.Equal(t, tc.expected, multiBidMap, tc.desc)
```
[the remark this answers is not in the document]
Comment (guscarreon) [reply exchange/exchange_test.go]: done

File: exchange/exchange_test.go (hunk 2 above)
Comment (guscarreon) [reply exchange/exchange_test.go:5759]: Good catch! That extra test revealed another spot where case insensitivity was needed inside function `buildRequestExtMultiBid()`. Added `exchange/exchangetest/multi-bids-mixed-case.json`.

File: exchange/exchange_test.go [hunk 4]
```diff
@@ -2259,8 +2259,13 @@ func TestTimeoutComputation(t *testing.T) {
 func TestExchangeJSON(t *testing.T) {
 	if specFiles, err := os.ReadDir("./exchangetest"); err == nil {
 		for _, specFile := range specFiles {
+			if !strings.HasSuffix(specFile.Name(), ".json") {
```
Comment (guscarreon) [inline exchange/exchange_test.go:2262]: Added so vim's `.swp` files wouldn't error. Can remove if you want

File: exchange/utils.go [hunk 5]
```diff
@@ -467,12 +467,12 @@ func buildRequestExtMultiBid(adapter string, reqMultiBid []*openrtb_ext.ExtMulti
 	adapterMultiBid := make([]*openrtb_ext.ExtMultiBid, 0)
 	for _, multiBid := range reqMultiBid {
 		if multiBid.Bidder != "" {
-			if multiBid.Bidder == adapter || isBidderInExtAlternateBidderCodes(adapter, multiBid.Bidder, adapterABC) {
+			if strings.ToLower(multiBid.Bidder) == adapter || isBidderInExtAlternateBidderCodes(adapter, strings.ToLower(multiBid.Bidder), adapterABC) {
```
Comment (guscarreon) [inline exchange/utils.go:470]: If we use `openrtb_ext.NormalizeBidderName()` here, `exchange/exchangetest/multi-bids-with-extra-bids.json` fails. Whether or not the expectations in said test are correct we can leave for a future discussion.

**Review by VeronikaSolovei9 — APPROVED** [review]
LGTM

**Review by AlexBVolcy — APPROVED** [review]

**Pull request merged**